docs/UserGuide.md

```diff
@@ -152,7 +173,7 @@ Format: `add-class n/CLASS_NAME`
 Examples:
 * `add-class n/CS2103T Tutorial T10`
 
-#### Listing all students within a class : `list-students`
+#### List Students In A Class : `list-students`
```

Comment:
`In` and `A` shouldn't be capitalised in the title.

Reply:
I thought of that, but the issue is that the formatting will not be standardized across the table and commands. I feel that perhaps its better to caps everything so that we standardize our documentation

Reply:
Will be keeping line 155 instead